web/packages/teleterm/src/ui/services/workspacesService/profileColor.ts

```diff
@@ -1,5 +1,7 @@
 import { z } from 'zod';
 
+import { dataVisualisationColors } from 'design/theme/themes/darkTheme';
```

Comment:
Let's add a comment to the definition of `dataVisualisationColors` in the dark theme that those colors are used for the user icon in Connect. Otherwise what might seem like a minor change might end up being a huge change for users. I'd like the person who's going to modify `dataVisualisationColors` to be aware of this.